Birthday/ForgotPassword/ForgotPasswordView.swift

```diff
@@ -0,0 +1,160 @@
+import SwiftUI
+
+struct ForgotPasswordView: View {
+  @StateObject private var viewModel = ForgotPasswordViewModel(
+    signInRepository: SignInRepositoryImpl()
+  )
+  @State private var isVerificationCodeViewVisible = false
+  @State private var codeText = ""
+  @State private var isPasswordHidden = true
+  
+  private var isCodeMatched: Bool {
```

Comment:
in my opinion you can move this to VM as a parametrized method. Please check it!